### PR TITLE
feature_PRESIDECMS-1252 ::

### PR DESCRIPTION
**Pull request by cfmitrah**
- Fixed UI overlaps issue.

### CODE REVIEW

File: system/views/admin/rulesEngine/index.cfm [hunk 1]
```diff
@@ -5,8 +5,9 @@
 
 
 <cfoutput>
-	<div class="top-right-button-group">
+	<div class="top-right-button-group" style="margin: -8px 4px 0px 0px;">
```
Comment (DominicWatson) [inline system/views/admin/rulesEngine/index.cfm]: Not sure on this one. I think we need a global fix for this issue with the 'top-right-button-group'. i.e. some kind of `event.addAdminTopRightButton()`, `event.getAdminTopRightButtons()` set of methods. These could then be output at the very end of the request in a CSS layout that doesn't require absolute positioning.

Can you remove this bit from the PR and we can then merge the other change?

**Review by DominicWatson — CHANGES_REQUESTED** [review]
Thanks @cfmitrah - please see review comments.

**Comment by cfmitrah** [comment]
> Can you remove this bit from the PR and we can then merge the other change?

Yes I removed that inline style, now  you can merge the other change.

**Comment by DominicWatson** [comment]
Garrrr, this got auto closed because the target branch has disappeared. Really sorry for this. You should be able to re-open from your branch though by targeting the `stable` branch.

**Pull request closed without merging**